ee/clickhouse/migrations/0011_cohortpeople_no_shard.py

```diff
@@ -0,0 +1,7 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.sql.cohort import CREATE_COHORTPEOPLE_TABLE_SQL, DROP_COHORTPEOPLE_TABLE_SQL
+from posthog.settings import CLICKHOUSE_REPLICATION
+
+# run create table again with proper configuration
+operations = [migrations.RunSQL(DROP_COHORTPEOPLE_TABLE_SQL), migrations.RunSQL(CREATE_COHORTPEOPLE_TABLE_SQL)]
```

Comment:
Does this need an `ON CLUSTER posthog` clause?

Reply:
Not yet - https://github.com/PostHog/posthog/issues/4415 will handle that. Worried that e.g. charts won't have the same cluster name and need to make it configurable :)